src/main/java/com/hms_networks/americas/sc/extensions/system/http/SCHttpUtility.java

```diff
@@ -11,7 +11,7 @@
  * Utility class for performing and managing HTTP connections on the Ewon Flexy system.
  *
  * @since 1.3.0
- * @version 1.0.0
+ * @version 1.0.1
```

Comment:
How are we handling version updates to files. I can fixup w/ previous but if there were multiple changes that required multiple commits prior to this I would expect this to be a seperate commit so im inclined to think its most proper in its own commit to indicate this is the state of v1.0.1.

Reply:
I've mainly been doing that to avoid frequently incrementing the class-wide @ version tag. That being said, for classes with an already established versioning scheme, I am ok with just sticking to that.

Reply:
will fix

Reply:
Fixed